.travis/run_tests.sh

```diff
@@ -7,14 +7,6 @@ cd build
 cmake -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DPYTHON_EXECUTABLE=/usr/bin/python3 -DCMAKE_INSTALL_PREFIX=../install ..
 make -j2 install
 
-# Units tests
-./units/test/scipp-units-test
-
-# Core tests
-./core/test/scipp-core-test
```

Comment:
I had chosen to run tests directly, since it is "really" slow with ctest.

Reply:
I didn't realise it was slower!